common/table.js

```diff
@@ -494,34 +494,61 @@
             }
         }
 
+        function _callonSelectedRowsChanged (scope, tuples, isSelected) {
+            if (scope.onSelectedRowsChangedBind) {
+                return scope.onSelectedRowsChangedBind()(tuples, isSelected);
+            } else if (scope.onSelectedRowsChanged) {
+                return scope.onSelectedRowsChanged()(tuples, isSelected);
+            }
+        }
+
+        /**
+         * Scrolls the current container to top. The container
+         * can be the modal-body or the main-container
+         */
+        function scrollToTop() {
```

Comment:
This shouldn't necessarily be refactored here, but now we have multiple places with a `scrollToTop` function. We could change this to be a utility function that takes in the element you want to scroll to the top of.

Reply:
I could leave this in the place that it is being called. The ` $window.scrollTo(0, 0);` wasn't doing anything since it's not the window that is scrollable but the container inside the window. I think I created the function because I thought it will be called multiple times.